rocketpy/AeroSurfaces.py

```diff
@@ -19,6 +19,16 @@ def __init__(self):
     def append(self, aeroSurface, position):
         self._aeroSurfaces.append((aeroSurface, position))
 
+    def remove(self, aeroSurface):
+        for surface, position in self._aeroSurfaces:
+            if surface == aeroSurface:
+                self._aeroSurfaces.remove((aeroSurface, position))
+
+    def pop(self, aeroSurface):
+        for surface, position in self._aeroSurfaces:
+            if surface == aeroSurface:
+                self._aeroSurfaces.pop((aeroSurface, position))
```

Comment:
I am quite confused about this method. The list `pop` method takes a single input, the index of the item to be removed, but here, a tuple is given. By default, if no index is given, -1 is used (i.e. the last item of the list is removed and returned).

Here is a possible implementation:
```suggestion
    def pop(self, index=-1):
        return self._aerosurfaces.pop(index)
```

This way, the method can be used to remove the last aerodynamic surface added. Since it is returned, it can be modified and later on be added or appended again.